stock_move_backdating/wizard/stock_partial_picking_view.xml

```diff
@@ -8,7 +8,9 @@
             <field name="inherit_id" ref="stock.stock_partial_picking_form" />
             <field name="arch" type="xml">
                 <xpath expr="//field[@name='move_ids']/tree//field[@name='product_uom']" position="after">
-                    <field name="date_backdating" on_change="on_change_date_backdating(date_backdating)"/>
+                    <field
```

Comment:
I prefer to put to save one line and see more indented the attributes (more than a new tag):
```
<field name="date_backdating" 
       on_change="on_change_date_backdating(date_backdating)"/>
```